src/main/kotlin/com/pubnub/api/endpoints/presence/SetState.kt

```diff
@@ -48,6 +49,8 @@ class SetState internal constructor(
 
         addQueryParams(queryParams)
 
+        PubNubUtil.maybeAddEeQueryParam(pubnub.configuration, queryParams)
```

Comment:
I would move this to addQueryParams method that is responsible for adding query params.
What do you think?

Reply:
do we want "ee" in SetState in the end?

Reply:
I think we need to wait with this till decision will be made. I would merge changes as they are(without ee in setState) and create story regarding this remaining case. Once decision is made we can add ee into setState or leave it as is i.e. without ee.

Reply:
sounds good, done.